quick-build.md

```diff
@@ -77,3 +77,15 @@ To read hilog output:
 ```bash
 hdc hilog
 ```
+
+### Speeding Up Build Times
+
+Experiencing prolonged build times can be frustrating. A practical solution to mitigate this issue involves optimizing the use of Docker containers and caching mechanisms. By mounting directories for prebuilts and ccache when initiating the Docker container, you can significantly reduce build times for subsequent builds. This approach ensures that once the prebuilts are downloaded, they don't need to be fetched again, and the compilation cache is maintained across builds.
+
+To apply this optimization, use the following command to start your Docker container:
+
+```bash
+docker run -it -v $(pwd):/home/openharmony/workdir -v ~/openharmony_prebuilts:/home/openharmony/openharmony_prebuilts -v ~/.ccache:/root/.ccache swr.cn-south-1.myhuaweicloud.com/openharmony-docker/docker_oh_standard:3.2
+```
+
+After starting the container with the above command, navigate to the `workdir` directory before initiating the build process.

```

Comment:
For most readers this will be obvious, but for completeness, I'd add a command in the documentation showing how to enter the `workdir` directory.

Reply:
done